<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QTalarm</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/new/icons/Clock.png</normaloff>:/new/icons/Clock.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0">
     <widget class="QCheckBox" name="chkThurs">
      <property name="text">
       <string>Thursday</string>
      </property>
     </widget>
    </item>
    <item row="16" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Time:</string>
      </property>
     </widget>
    </item>
    <item row="19" column="0">
     <widget class="QLabel" name="txtSoundPath">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="chkTues">
      <property name="text">
       <string>Tuesday</string>
      </property>
     </widget>
    </item>
    <item row="17" column="2" colspan="2">
     <widget class="QDateEdit" name="CustEdit"/>
    </item>
    <item row="17" column="0" colspan="2">
     <widget class="QCheckBox" name="chkCustom">
      <property name="text">
       <string>Custom Date:</string>
      </property>
     </widget>
    </item>
    <item row="21" column="0" colspan="8">
     <widget class="QListWidget" name="listWidget">
      <property name="maximumSize">
       <size>
        <width>1677720</width>
        <height>1677720</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="18" column="0">
     <widget class="QCheckBox" name="chkSounds">
      <property name="text">
       <string>Use File</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QCheckBox" name="chkSun">
      <property name="text">
       <string>Sunday</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="chkWed">
      <property name="text">
       <string>Wednesday</string>
      </property>
     </widget>
    </item>
    <item row="21" column="8">
     <widget class="QDialogButtonBox" name="listAlmBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
    <item row="16" column="2" colspan="2">
     <widget class="QTimeEdit" name="timeEdit">
      <property name="displayFormat">
       <string>hh:mm:ss:AP</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="chkMon">
      <property name="text">
       <string>Monday</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QCheckBox" name="chkFri">
      <property name="text">
       <string>Friday</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QCheckBox" name="chkSat">
      <property name="text">
       <string>Saturday</string>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QGroupBox" name="groupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Current Time:</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="Clock">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>26</pointsize>
          </font>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="7" column="5">
     <widget class="QPushButton" name="TestBtn">
      <property name="text">
       <string>Test</string>
      </property>
     </widget>
    </item>
    <item row="8" column="5">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Master Volume:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="5">
     <widget class="QSlider" name="VolumeSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>639</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Abo&amp;ut</string>
    </property>
    <addaction name="actionAbout_QT"/>
    <addaction name="actionAbout_QTalam"/>
   </widget>
   <widget class="QMenu" name="menublotenko">
    <property name="title">
     <string>blotenko</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
   <addaction name="menublotenko"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionAbout_QT">
   <property name="text">
    <string>&amp;About QT</string>
   </property>
  </action>
  <action name="actionAbout_QTalam">
   <property name="text">
    <string>About &amp;QTalam</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
